tests/data/ext/sample-status-invalid-format.json

```diff
@@ -0,0 +1,36 @@
+[
```

Comment:
can we add a comment somewhere in the file to explain what is wrong with it? maybe a string in caps as part of the message? for an example, see tests/data/events/custom_script_invalid_json.tld

Reply:
same for the other files

Reply:
Done.

Reply:
Renamed the files to point out the issues.